scripts/ci/update-translations.sh

```diff
@@ -4,9 +4,8 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source ${DIR}/../version_number.sh
 
 lupdate QField.pro
+
+# release only if the branch is master, no longer transifex "release-*"
```

Comment:
```suggestion
# release only if the branch is master
```

No reason to comment historical considerations

Reply:
clear